get_coverage_for_challenge.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+challengeId=$1
+JACOCO_TEST_REPORT_CSV_FILE="${SCRIPT_CURRENT_DIR}/build/reports/jacoco/test/jacocoTestReport.csv"
```

Comment:
The script now needs to be updated.

@neomatrix369 to do:
- [ ] parse `.xml` using `xmllint`
- [ ] compute the coverage as a percentage: `covered * 100 / (covered + missed)`
- [ ] the stdout should only contain the coverage, stderr to be used for anything else